### ISSUE TITLE
Issues with BigInt and Safari 14.

### ISSUE THREAD

**Issue by pur3miish** [amilajack/eslint-plugin-compat#457]
### Issue

Safari 14 supports [BigInteger](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) API.
I am however receiving eslint errors that it is unsupported.

Here is my browsers list config in `package.json`, after running 
```shell
$ npx browserslist
```
I see the following
<img width="374" alt="Screen Shot 2021-03-25 at 10 46 22 pm" src="https://user-images.githubusercontent.com/16223777/112475015-0f00a280-8dbc-11eb-8cc9-3658601f3396.png">

Here are my dependencies.
<img width="371" alt="Screen Shot 2021-03-25 at 10 49 03 pm" src="https://user-images.githubusercontent.com/16223777/112475235-4ff8b700-8dbc-11eb-9823-69f9ba9c8689.png">

### Expected behaviour 

After adding `supports bigint` to config list I expected to have no eslint errors.
However after running eslint in terminal I receive the following errors.
<img width="553" alt="Screen Shot 2021-03-25 at 10 54 59 pm" src="https://user-images.githubusercontent.com/16223777/112475970-21c7a700-8dbd-11eb-9185-cff1f0ab6bd3.png">

But from what I can discern BigInts are supported in [safari 14](https://caniuse.com/?search=bigInt).

Thanks.
